src/ui/public/agg_types/controls/percentiles.html

```diff
@@ -4,6 +4,7 @@
     ng-model="agg.params.percents"
     unit-name="percent"
     range="[0,100]"
+    validate-order="false"
```

Comment:
How about representing more of the semantics in the name, e.g. `validate-ascending-order`? Or we could make it more flexible by allowing different orderings as in `validate-order="asc|desc"`.